lumen/ai/actor.py

```diff
@@ -11,6 +11,8 @@
 from panel.chat import ChatFeed
 from pydantic import BaseModel
 
+from lumen.ai.config import PROMPTS_DIR
```

Comment:
Please be consistent in the imports. I know relative imports aren't everyone's cup of tea but as long as we're using them let's be consistent about it.

Reply:
Turns out we were very inconsistent about it heh
https://github.com/holoviz/lumen/pull/1187/commits/2edc698cc8fe11fb85a1d6c5cb8559625c4a6f7a